metadata-io/src/main/java/com/linkedin/metadata/search/aggregator/AllEntitiesSearchAggregator.java

```diff
@@ -57,6 +61,7 @@ public AllEntitiesSearchAggregator(
     _searchRanker = Objects.requireNonNull(searchRanker);
     _cachingEntitySearchService = Objects.requireNonNull(cachingEntitySearchService);
     _entityDocCountCache = new EntityDocCountCache(entityRegistry, entitySearchService);
+    _entityRegistry = entityRegistry;
```

Comment:
why add `entityRegistry` here if it's not being used? Same thing with the two imports above